inspire/modules/workflows/tasks/submission.py

```diff
@@ -91,6 +91,7 @@ def _create_ticket(obj, eng):
             email=email,
             title=d.title,
             identifier=sip.metadata.get("system_number_external", {}).get("value", ""),
+            user_comment=sip.metadata.get("hidden_note", {}).get("value", ""),
```

Comment:
That's fine.

```python
hidden_note:
    schema:
        {'hidden_note': {'type': 'list', 'force': True}}
```

should be added as well as far as I can see.

By the way,

```python
user_comment=filter(lambda k: 'submitter' == k['source'],
                    sip.metadata.get("hidden_note", []))
```

will throw ```KeyError: 'source'``` if one of the elements in the ``hidden_note`` does not contain the ``source`` key. I guess that might be the case in the future if we add some extra hidden notes?